spec/controllers/verify/sessions_controller_spec.rb

```diff
@@ -245,5 +245,26 @@
         end
       end
     end
+
+    describe '#destroy' do
+      it 'clears the idv session' do
+        delete :destroy
+
+        expect(controller.user_session[:idv]).to eq({})
+      end
+
+      it 'redirects user to profile' do
+        session = { idv: {}, first_time_recovery_code_view: true }
+        allow(controller).to receive(:user_session).and_return(session)
+        delete :destroy
+        expect(response).to redirect_to(profile_path)
+      end
+
+      it 'redirects user to recovery code page' do
+        delete :destroy
+        subject.user_session[:first_time_recovery_code_view] = false
+        expect(response).to redirect_to(manage_recovery_code_path)
+      end
```

Comment:
Wrap these inside 2 different contexts? That way, it's clear that there are 2 scenarios: one where the user already has a recovery code, and one where they don't. Also, the setup in this test seems different. Shouldn't they both set the session the same way, and do it as the first step, then call `delete :destroy`?